exercises/handling-get-requests/app.rb

```diff
@@ -86,7 +86,7 @@ def comparable_author(author)
 
 def quotes_by(author)
```

Comment:
Although you've defined a `QUOTES` constant, in a method like this I'd pass it in as an argument regardless, e.g.,

```ruby
def quotes_by(quotes, author)
  quotes.select { |quote| comparable_author(quote[:author]) == comparable_author(author) }
end
```

Referencing the `QUOTES` constant directly creates coupling between this method and the rest of your code.  If you ever define your quotes in some other way or get them from some place, you'll have to remember to change this method.